similarity-data-impl/src/main/java/org/phenotips/data/similarity/script/PatientMatchingScriptService.java

```diff
@@ -55,7 +59,7 @@
     public void clearCache()
     {
         ((DefaultPatientSimilarityViewFactory) patientViewFactory).clearCache();
-        PatientGenotype.clearCache();
+        ((ExomiserExomeManager) exomeManager).clearCache();
```

Comment:
I considered that, but figured it was too implementation-specific to be included in the interface. Same with the `PatientSimilarityViewFactory`. :/